components/style/gecko/wrapper.rs

```diff
@@ -1136,6 +1146,68 @@ impl<'le> ::selectors::Element for GeckoElement<'le> {
         None
     }
 
+    fn attr_matches(&self,
+                    ns: &NamespaceConstraint<&Namespace>,
+                    local_name: &Atom,
+                    operation: &AttrSelectorOperation<&Atom>)
+                    -> bool {
+        unsafe {
+            match *operation {
+                AttrSelectorOperation::Exists => {
```

Comment:
Ok. Unless you this this is important to do soon I’d prefer to leave it to do later, since this PR is in the queue now :) Another idea is to have an `operator` parameter to cut the number of FFI functions.